src/main/java/uk/gov/hmcts/reform/slc/services/servicebus/exceptions/ConnectionException.java

```diff
@@ -1,7 +1,12 @@
 package uk.gov.hmcts.reform.slc.services.servicebus.exceptions;
 
-public class ConnectionException extends RuntimeException {
+import uk.gov.hmcts.reform.logging.exception.AbstractLoggingException;
+import uk.gov.hmcts.reform.logging.exception.AlertLevel;
+
+import static uk.gov.hmcts.reform.slc.logging.ErrorCode.UNKNOWN;
+
+public class ConnectionException extends AbstractLoggingException {
     public ConnectionException(String message, Throwable cause) {
-        super(message, cause);
+        super(AlertLevel.P1, UNKNOWN, message, cause);
```

Comment:
What does P1,P2.. signifies?

Reply:
Business way of grouping exceptions to alert level priorities and respond accordingly. Depending on terms and/or conditions, each priority has required response time starting from P1 as being up most highest priority